src/Core/Models/Data/SelectionReadOnly.cs

```diff
@@ -6,5 +6,6 @@ public class SelectionReadOnly
     {
```

Comment:
Wonder if we shouldn't rename this class now that I think about it more since it's no longer scoped to just `ReadOnly` but basically 1 or more permissable attributes. @kspearrin thoughts? We could do it in a future iteration also if necessary.

Reply:
Yea, it spans all codebases. Let's circle back one day (which means we probably never will 😆 )